lib/adapters/autocomplete-adapter.ts

```diff
@@ -62,31 +63,42 @@ export default class AutocompleteAdapter {
     const triggerChars =
       server.capabilities.completionProvider != null ?
         server.capabilities.completionProvider.triggerCharacters || [] : [];
-    const triggerPoint = AutocompleteAdapter.getTriggerPoint(request, triggerChars);
-    const prefixWithTrigger = AutocompleteAdapter.getPrefixWithTrigger(request, triggerPoint);
+    const triggerChar = AutocompleteAdapter.getTriggerCharacter(request, triggerChars);
+    const prefixWithTrigger = triggerChar + request.prefix;
+    const triggerColumn = request.bufferPosition.column - prefixWithTrigger.length;
+    const triggerPoint = new Point(request.bufferPosition.row, triggerColumn);
+
+    // Only auto-trigger on a trigger character or after the minimum number of characters from autocomplete-plus
+    const minimumWordLength = atom.config.get('autocomplete-plus.minimumWordLength') || 0;
```

Comment:
This couples atom-languageclient to autocomplete-plus, is there a way we can have the client register a method we can call to get the current word length?